File: CS/Form1.cs
using System;
using System.Windows.Forms;
using System.Globalization;
using System.Threading;

namespace SchedulerSwitchLocale {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
            cbLocale.SelectedIndex = cbLocale.Items.IndexOf(CultureSwitcher.CultureId);
        }

        private void cbLocale_SelectedIndexChanged(object sender, EventArgs e) {
            string selectedLocale = cbLocale.SelectedItem.ToString();
            CultureSwitcher.ChangeCulture(selectedLocale);
        }
    }
}

File: VB/Form1.vb
Imports System
Imports System.Windows.Forms

Namespace SchedulerSwitchLocale

    Public Partial Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
            cbLocale.SelectedIndex = cbLocale.Items.IndexOf(CultureId)
        End Sub

        Private Sub cbLocale_SelectedIndexChanged(ByVal sender As Object, ByVal e As EventArgs)
            Dim selectedLocale As String = cbLocale.SelectedItem.ToString()
            ChangeCulture(selectedLocale)
        End Sub
    End Class
End Namespace
